pontoon/api/serializers.py

```diff
@@ -27,25 +55,25 @@ class Meta:
         model = Locale
         fields = [
             "code",
-            "approved_strings",
-            "cldr_plurals",
-            "complete",
+            "name",
             "direction",
+            "population",
+            "cldr_plurals",
+            "plural_rule",
+            "script",
             "google_translate_code",
-            "missing_strings",
             "ms_terminology_code",
             "ms_translator_code",
-            "name",
-            "plural_rule",
-            "population",
-            "pretranslated_strings",
-            "script",
-            "strings_with_errors",
-            "strings_with_warnings",
             "systran_translate_code",
             "team_description",
             "total_strings",
+            "pretranslated_strings",
```

Comment:
Let's use the same order as in the dashboards (and `ProjectLocaleSerializer`): `approved_strings` before `approved_strings`.